### PR TITLE
tests: Test all rpc commands in rpc_platform_filter.py

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: test/functional/rpc_platform_filter.py [hunk 1]
```diff
@@ -62,11 +78,15 @@ def test_command(method, params, auth, expexted_status):
         test_command("getblockcount", [], rpcuser_authpair_platform, 200)
         test_command("getbestchainlock", [], rpcuser_authpair_platform, 500)
 
-        self.log.info('Try running a not whitelisted command...')
-        test_command("stop", [], rpcuser_authpair_platform, 403)
+        self.log.info('Try running all non-whitelisted commands as each user...')
+        for command in nonwhitelisted:
+            test_command(command, [], rpcuser_authpair_platform, 403)
+            if command != "stop": # avoid stoping the node while testing
```
Comment (xdustinface) [inline test/functional/rpc_platform_filter.py:84]:
```suggestion
            if command != "stop":  # avoid stoping the node while testing
```
Just a python thing

**Review by xdustinface — CHANGES_REQUESTED** [review]
Looks good, just two nits below.

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**